composer.json

```diff
@@ -252,7 +252,8 @@
                 "3250298 - Return empty string '' with JSON Serializer instead of FALSE": "https://www.drupal.org/files/issues/2024-08-05/3250298-17.patch",
                 "2313309 - Admin toolbar and contextual links should always be rendered in the admin language (if set)": "https://www.drupal.org/files/issues/2023-12-19/2313309-179.patch",
                 "3054641 - Local tasks should honor selected admin interface language (if set)": "https://www.drupal.org/files/issues/2024-06-21/3054641-84.patch",
-                "3410100 - [10.2 regression] CKEditor 5 breaks when 'Source'/Source editing button is added and 'Manually editable HTML tags' are specified": "https://www.drupal.org/files/issues/2024-01-23/drupal-revert-source-editing-validation-tightening-3410100-38.patch"
+                "3410100 - [10.2 regression] CKEditor 5 breaks when 'Source'/Source editing button is added and 'Manually editable HTML tags' are specified": "https://www.drupal.org/files/issues/2024-01-23/drupal-revert-source-editing-validation-tightening-3410100-38.patch",
+                "3381979 More link is missing in pager ": "https://www.drupal.org/files/issues/2024-08-03/drupal-3381979-32.patch"
```

Comment:
Unless this is a very special case, a patch should have changed the composer.lock file as well. Could you include the lock file?